pkg/monitoring/domainstats/downwardmetrics/scraper.go

```diff
@@ -55,6 +51,17 @@ func (s *Scraper) Scrape(socketFile string, vmi *k6sv1.VirtualMachineInstance) {
 	}
 	defer cli.Close()
 
+	version, err := cli.GetQemuVersion()
+
+	if err != nil {
+		if cmdclient.IsUnimplemented(err) {
```

Comment:
up to you if you prefer setting something like `unknown` as the detected version if we have this case or empy,